docs/bridge/docs/04-Routers/RFQ/API/index.md

```
@@ -60,6 +60,54 @@ Please note that end-users and solvers will not need to run their own version of
+  The websocket API exposes several operations for quoters:
+  - `ping` - sends a heartbeat to the API server to keep the connection alive (must be sent at least once per minute)
+  - `subscribe` - subscribes to quote requests for given chain(s)
+  - `unsubscribe` - unsubscribes to quote requests for given chain(s)
+  - `send_quote` - responds to a quote request
+
+  The API server may respond with the following operations:
+  - `pong` - acknowleges a `ping` message
+  - `request_quote` - informs quoter of a new user quote request
+
+  All websocket messages follow this format:
+  ```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)